### PR TITLE
add description in docker plugin ls

### PR DESCRIPTION
**Pull request by vieux**
```
$ docker plugin ls
NAME                  TAG                 DESCRIPTION                ACTIVE
tiborvass/no-remove   latest              A test plugin for Docker   true
vieux/sshfs           latest              sshFS plugin for Docker    true
```

**\- Description for the changelog**
add description in `docker plugin ls`

**\- A picture of a cute animal (not mandatory but encouraged)**
maxresdefault

@tiborvass @anusha-ragunathan

### CODE REVIEW

File: api/client/plugin/list.go [hunk 1]
```diff
@@ -33,11 +33,11 @@ func runList(dockerCli *client.DockerCli) error {
 	}
 
 	w := tabwriter.NewWriter(dockerCli.Out(), 20, 1, 3, ' ', 0)
-	fmt.Fprintf(w, "NAME \tTAG \tACTIVE")
+	fmt.Fprintf(w, "NAME \tTAG \tDESCRIPTION\tACTIVE")
 	fmt.Fprintf(w, "\n")
 
 	for _, p := range plugins {
-		fmt.Fprintf(w, "%s\t%s\t%v\n", p.Name, p.Tag, p.Active)
+		fmt.Fprintf(w, "%s\t%s\t%s\t%v\n", p.Name, p.Tag, p.Manifest.Description, p.Active)
```
Comment (icecrime) [inline api/client/plugin/list.go]: Should we limit in size and append `...` over a given length?

**Comment by vdemeester** [comment]
Design LGTM with @icecrime comment :angel:

File: api/client/plugin/list.go (hunk 1 above)
Comment (thaJeztah) [inline api/client/plugin/list.go]: This is the code used in `docker search`; [api/client/image/search.go#L111-L114](https://github.com/docker/docker/blob/a502158d8a9b3a3949eb0bc0b087138ef65efabb/api/client/image/search.go#L111-L114)

perhaps we should enhance the `stringutils.Truncate` to take an extra parameter for an ellipsis

**Comment by vieux** [comment]
done PTAL

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by thaJeztah** [comment]
hm, flaky test? https://jenkins.dockerproject.org/job/Docker-PRs-experimental/22261/console

```
20:48:41 ----------------------------------------------------------------------
20:48:41 FAIL: docker_api_swarm_test.go:251: DockerSwarmSuite.TestApiSwarmServicesCreateGlobal
20:48:41 
20:48:41 [d96215372] waiting for daemon to start
20:48:41 [d96215372] daemon started
20:48:41 [d73393837] waiting for daemon to start
20:48:41 [d73393837] daemon started
20:48:41 [d90007875] waiting for daemon to start
20:48:41 [d90007875] daemon started
20:48:41 [d96553713] waiting for daemon to start
20:48:41 [d96553713] daemon started
20:48:41 [d21800249] waiting for daemon to start
20:48:41 [d21800249] daemon started
20:48:41 docker_api_swarm_test.go:265:
20:48:41     waitAndAssert(c, defaultReconciliationTimeout, d4.checkActiveContainerCount, checker.Equals, 1)
20:48:41 github.com/docker/docker/integration-cli/_test/_obj_test/docker_utils.go:2011:
20:48:41     ...open github.com/docker/docker/integration-cli/_test/_obj_test/docker_utils.go: no such file or directory
20:48:41 ... obtained int = 0
20:48:41 ... expected int = 1
20:48:41 
20:48:41 [d96215372] exiting daemon
20:48:41 [d73393837] exiting daemon
20:48:41 [d90007875] exiting daemon
20:48:41 [d96553713] exiting daemon
20:48:41 [d21800249] exiting daemon
20:48:43 
20:48:43 ----------------------------------------------------------------------
```

**Comment by vdemeester** [comment]
LGTM 👼

**Comment by thaJeztah** [comment]
docs LGTM

**Comment by vdemeester** [comment]
docs LGTM 🐸

**Pull request merged**